torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -211,6 +215,17 @@ std::string shapesToStr(const std::vector<std::vector<int64_t>>& shapes) {
   return oss.str();
 }
 
+std::string stacksToStr(const std::vector<std::string>& stacks) {
+  std::ostringstream oss;
+  for (auto idx = 0; idx < stacks.size(); ++idx) {
```

Comment:
please fix clang-tidy here

Reply:
> please fix clang-tidy here

This strictly follows the code style in the above function "shapesToStr". So I wonder if using "auto" is better than the long string "std::vector<std::string>::size_type".

Reply:
Thanks. Done.

Reply:
it can be just `size_t`